package.json

```diff
@@ -22,16 +22,25 @@
     "index.js"
   ],
   "scripts": {
-    "lint": "jshint index.js test/",
-    "unit": "node test/index.js && node test/read.js && node test/windows.js",
+    "fmt": "prettier --write index.js test/",
+    "lint": "eslint index.js test/",
+    "unit": "jest",
     "test": "npm run lint && npm run unit"
```

Comment:
Can you use this approach?

https://github.com/lydell/js-tokens/blob/417dc4085a7e3468d01a6371abc81373188dbc43/package.json#L5-L9

It requires adding stuff to .prettierignore and .eslintignore as needed.

Reply:
Do you prefer blacklist files instead of whitelisting what needs to be linted/formatted?

Reply:
Yes!